src/DIRAC/Core/Tornado/Server/HandlerManager.py

```diff
@@ -103,7 +101,7 @@ def __load(self, instances, componentType, pathFinder):
         # Extract ports, e.g.: ['Framework/MyService', 'Framework/MyService2:9443]
         port, instances = self.__extractPorts(instances)
 
-        loader = ModuleLoader(componentType, pathFinder, RequestHandler, moduleSuffix="Handler")
+        loader = ModuleLoader(componentType, pathFinder, False, moduleSuffix="Handler")
```

Comment:
Can you please explain why you drop the check that the handler class should inherit from the RequestHandler ?

Reply:
I dropped it simply because I did not find it useful.

Reply:
Then this is also true for other components (Agents). So, this can be removed from the ModuleLoader at all.

Reply:
Indeed. Removed.